core/src/main/resources/error/error-classes.json

```diff
@@ -1754,6 +1779,11 @@
     ],
     "sqlState" : "42826"
   },
+  "OPERATION_NOT_ALLOWED" : {
+    "message" : [
+      "Operation not allowed: <message>. This error occurs when attempting an operation that is not currently supported. Please check the documentation for the list of allowable operations."
```

Comment:
Instead of the generic `message`. Please, add sub-classes.

I told you about this in your PR https://github.com/apache/spark/pull/39965

Reply:
Oh,,, I didn't realized that the PR was closed. Let me revisit my PR and revert the change here in this PR.